openpgp/packet/aead_config.go

```diff
@@ -8,7 +8,7 @@ import "math/bits"
 // A nil AEADConfig is valid and results in all default values.
 type AEADConfig struct {
 	// The AEAD mode of operation.
-	DefaultMode AEADMode
+	DefaultMode         AEADMode
```

Comment:
Why adding whitespace?

Reply:
That was `gofmt` in the previous commit (with `AEADModePreferences`). Will fix this

Reply:
Done :+1: